src/app/(admin)/admin/settings/rooms/page.tsx

```diff
@@ -6,14 +6,45 @@ import Link from "next/link";
 import { api } from "~/trpc/react";
 
 const Rooms = () => {
+  const { data: rooms, isLoading } = api.rooms.list.useQuery();
   const router = useRouter();
+  console.log(rooms)
```

Comment:
Let's remove this log to keep the console clean

Reply:
Done